nvm.sh

```diff
@@ -146,7 +146,20 @@ nvm_has_system_iojs() {
 }
 
 nvm_is_version_installed() {
-  [ -n "${1-}" ] && [ -x "$(nvm_version_path "$1" 2>/dev/null)"/bin/node ]
+  if [ -z "${1-}" ]; then
+    return 1
+  fi
+  local VERSION_PATH
+  VERSION_PATH="$(nvm_version_path "$1" 2>/dev/null)"
+  if [ -x "$VERSION_PATH"/bin/node ]; then
+    return 0
+  fi
+  local NVM_OS
+  NVM_OS="$(nvm_get_os)"
+  if [ "_${NVM_OS}" = '_win' ] && [ -x "$VERSION_PATH"/bin/node.exe ]; then
+    return 0
+  fi
```

Comment:
Could be:
if [ -x "$VERSION_PATH"/bin/node ] || [ -x "$VERSION_PATH"/bin/node.exe ]; then